src/aat/resources/features/F-040/F-040.feature

```diff
@@ -0,0 +1,21 @@
+@F-040
+Feature: F-040: Get Case for Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-576
+  Scenario: must return successfully all and only the respective fields with READ access for Solicitors
+    Given a user with [an active Solicitor profile in CCD with a specific variation of ACLs on a case type]
+    And a user with [another active solicitor profile with different variation of ACLs on the same case type]
+    And a successful call [to create a token for case creation] as in [Befta_Jurisdiction2_Default_Token_Creation_Data_For_Case_Creation]
+    And another successful call [by a privileged user with full ACL to create a case of this case type] as in [Befta_Jurisdiction2_Default_Full_Case_Creation_Data]
+    When a request is prepared with appropriate values
+    And the request [is prepared on behalf of the user creating the case above]
```

Comment:
should mention that it is the privileged user making the call, eg. `And the request [is made by the privileged user who just created the case]`

Reply:
Agreed. Updated the code.